app/controllers/helperMethods.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const jwt = require('jsonwebtoken');
+
+class HelperMethod {
```

Comment:
### Critical

The whole essence of a `Helper Class` was/is to abstract functionality common to all classes i.e
-   are repeated in different Classes/Controllers
-   logically, should not belong to any one class

into one common location. 

The methods you have here do not meet any of the criteria stipulated above. There's no reason that these methods shouldn't exist in the Controllers that they service.

Please refactor accordingly.